pkg/tsdb/azuremonitor/metrics/azuremonitor-datasource.go

```diff
@@ -106,6 +106,7 @@ func (e *AzureMonitorDatasource) buildQueries(queries []backend.DataQuery, dsInf
 
 		timeGrain := azJSONModel.TimeGrain
 		timeGrains := azJSONModel.AllowedTimeGrainsMs
+		fmt.Println("TIME GRAINS ", timeGrains)
```

Comment:
this can be removed.

Reply:
missed that, will remove this and get a test in 😊